client/src/pages/Profile.js

```diff
@@ -0,0 +1,190 @@
+import React, { Component } from 'react';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import TextField from '@material-ui/core/TextField';
+import Box from '@material-ui/core/Box';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+// import { Route, Link } from "react-router-dom";
+import Card from '@material-ui/core/Card';
+import { withStyles } from "@material-ui/core/styles";
+import CardContent from '@material-ui/core/CardContent';
+import ProgressBar from '../components/LinearProgress';
+
+
+
+
+const useStyles = theme => ({
+    root: {
+        width: 700,
+        height: 500,
+        margin: "auto",
+        marginTop: 120,
+        borderRadius: "2%",
+        alignItems: 'center',
+
+    },
+    button: {
+        margin: theme.spacing(1),
+    },
+    box_Shadow: {
+        boxShadow: "0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19)"
+    },
+    paper: {
+        marginTop: theme.spacing(6),
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+    },
+    submit: {
+        margin: theme.spacing(3, 0, 2),
+        backgroundColor: "#FF7E00",
+        padding: 15,
+        paddingRight: 40,
+        paddingLeft: 40,
+        textTransform: "none",
+    },
+    center_Text: {
```

Comment:
is this a stylistic chose to name certain keys this way, where it's snake case and the second word is capitalized?

if not - i would make it stay consistent